lib/core/data-types/Factory.js

```diff
@@ -34,7 +37,7 @@ export default class DataTypeFactory {
     /**
      * Create a MantraDataType classed based on a type.
      * @param {object} params - Configuration object for the data type creation.
-     * @return {MantraDataType} A mantra data type object.
+     * @returns {MantraDataType} - A mantra data type object.
      */
     static create(params) {
```

Comment:
Remember to update the methods using the DATA_TYPES constant to the new way to declare it outside of the module scope.

Reply:
Sure! Will do.